docs/deployments/environments.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Environments / Instances
+description: An overview of the differences between Development and Production instances
+---
+
+# Environments / Instances
+When creating a new application within Clerk, you are provided with two instances: `Development` and `Production`.  These instances vary slightly and should only be used appropriately.
```

Comment:
we are labeling them as "environments" OR "instances" so can we include something like,

also, did we want to capitalize "development" and "production"?

```suggestion
When creating a new application within Clerk, you are provided with two **instances** (also known as **environments**): `development` and `production`.  These instances vary slightly and should only be used appropriately.
```

Reply:
I'd prefer we keep them capitalized because thats how we display them in the Dashboard